release_notes/ocp-4-21-release-notes.adoc

```diff
@@ -4,8 +4,194 @@
+
+// MachineOSConfig naming changes
+include::modules/rn-ocp-release-notes-mosc-naming.adoc[leveloffset=+2]
+
+// Deprecated and removed features
+include::modules/rn-ocp-release-notes-deprecated-removed-features.adoc[leveloffset=+1]
+
+// Images deprecated and removed features
+include::modules/rn-ocp-release-notes-images-dep-rem.adoc[leveloffset=+2]
+
+// Installation deprecated and removed features
+include::modules/rn-ocp-release-notes-images-dep-rem.adoc[leveloffset=+2]
+
+// Machine Management deprecated and removed features
+include::modules/rn-ocp-release-notes-machine-manage-dep-rem.adoc[leveloffset=+2]
+
+// Networking deprecated and removed features
+include::modules/rn-ocp-release-notes-networking-dep-rem.adoc[leveloffset=+2]
+
+// Node deprecated and removed features
+include::modules/rn-ocp-release-notes-node-dep-rem.adoc[leveloffset=+2]
+
+// OpenShift CLI (oc) deprecated and removed features
+include::modules/rn-ocp-release-notes-cli-dep-rem.adoc[leveloffset=+2]
+
+// Operator lifecycle and development deprecated and removed features
+include::modules/rn-ocp-release-notes-operators-dep-rem.adoc[leveloffset=+2]
+
+// Storage deprecated and removed features
+include::modules/rn-ocp-release-notes-storage-dep-rem.adoc[leveloffset=+2]
+
+//[id="ocp-clusters-dep-rem_{context}"]
+//=== Updating clusters deprecated and removed features
+
+//.Updating clusters deprecated and removed tracker
+//[cols="4,1,1,1",options="header"]
+//|====
+//|Feature |4.18 |4.19 |4.20
+//|====
+
+// Web console deprecated and removed features
+include::modules/rn-ocp-release-notes-web-console-dep-rem.adoc[leveloffset=+2]
+
+// Workloads deprecated and removed features
+include::modules/rn-ocp-release-notes-workloads-dep-rem.adoc[leveloffset=+2]
+
+// Deprecated features
+include::modules/rn-ocp-release-notes-deprecated-features.adoc[leveloffset=+1]
+
+// Removed features
+include::modules/rn-ocp-release-notes-removed-features.adoc[leveloffset=+1]
+
+// Bug fixes
+include::modules/rn-ocp-release-notes-bug-fixes.adoc[leveloffset=+1]
+
+
+[id="ocp-release-bug-fixes_{context}"]
+== Bug fixes
+//Bug fix work for TELCODOCS-750
+//Bare Metal Hardware Provisioning / OS Image Provider
+//Bare Metal Hardware Provisioning / baremetal-operator
+//Bare Metal Hardware Provisioning / cluster-baremetal-operator
+//Bare Metal Hardware Provisioning / ironic"
+//CNF Platform Validation
+//Cloud Native Events / Cloud Event Proxy
+//Cloud Native Events / Cloud Native Events
+//Cloud Native Events / Hardware Event Proxy
+//Cloud Native Events
+//Driver Toolkit
+//Installer / Assisted installer
+//Installer / OpenShift on Bare Metal IPI
+//Networking / ptp
+//Node Feature Discovery Operator
+//Performance Addon Operator
+//Telco Edge / HW Event Operator
+//Telco Edge / RAN
+//Telco Edge / Core
+
+
+//[id="ocp-release-note-cluster-override-admin-operator-bug-fixes_{context}"]
+//=== Cluster Resource Override Admission Operator
```

Comment:
Wouldn't all this be omitted since there is `modules/rn-ocp-release-notes-bug-fixes.adoc`?
```suggestion
```